<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1035</width>
    <height>361</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Datetime</string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="1" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Time:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="4">
         <widget class="QLineEdit" name="editTime">
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="1" column="5">
         <widget class="QPushButton" name="btnSetTime">
          <property name="text">
           <string>Set Time</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Date:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="3">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>0</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="2" column="4">
         <widget class="QLineEdit" name="editDate"/>
        </item>
        <item row="2" column="5" rowspan="2">
         <widget class="QPushButton" name="btnSetDate">
          <property name="text">
           <string>Set Date</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0" rowspan="2" colspan="2">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Datetime:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="2" rowspan="2">
         <widget class="QDateTimeEdit" name="dateTimeEdit">
          <property name="minimumSize">
           <size>
            <width>160</width>
            <height>0</height>
           </size>
          </property>
          <property name="displayFormat">
           <string>yyyy-MM-dd hh:mm:ss</string>
          </property>
         </widget>
        </item>
        <item row="4" column="4">
         <widget class="QLineEdit" name="editDatetime"/>
        </item>
        <item row="4" column="5">
         <widget class="QPushButton" name="btnSetDatetime">
          <property name="text">
           <string>Set Datetime</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QDateEdit" name="dateEdit">
          <property name="displayFormat">
           <string>yyyy-MM-dd</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QTimeEdit" name="timeEdit">
          <property name="displayFormat">
           <string>HH:mm:ss</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0" colspan="3">
         <widget class="QPushButton" name="btnGetTime">
          <property name="text">
           <string>GetCurrentDatetime</string>
          </property>
         </widget>
        </item>
        <item row="0" column="4">
         <widget class="QLabel" name="labelDatetime">
          <property name="text">
           <string>show</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Timer</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0" colspan="2">
           <widget class="QPushButton" name="btnStart">
            <property name="text">
             <string>Start</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2" colspan="2">
           <widget class="QPushButton" name="btnStop">
            <property name="text">
             <string>Stop</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Period:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="5">
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="5">
           <widget class="QLabel" name="labelElapsedTime">
            <property name="text">
             <string>elapsed time</string>
            </property>
           </widget>
          </item>
          <item row="1" column="4">
           <widget class="QPushButton" name="btnSetPeriod">
            <property name="text">
             <string>Set Period</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1" colspan="3">
           <widget class="QSpinBox" name="spinInterval">
            <property name="suffix">
             <string> ms</string>
            </property>
            <property name="maximum">
             <number>9999</number>
            </property>
            <property name="value">
             <number>1000</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLCDNumber" name="lcdHour"/>
          </item>
          <item>
           <widget class="QLCDNumber" name="lcdMin"/>
          </item>
          <item>
           <widget class="QLCDNumber" name="lcdSec"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QProgressBar" name="progressBar">
          <property name="value">
           <number>24</number>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="title">
        <string>Calendar</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Selected Date:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="editCalendar"/>
        </item>
        <item row="1" column="0" colspan="2">
         <widget class="QCalendarWidget" name="calendarWidget"/>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnClose">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>btnClose</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>575</x>
     <y>311</y>
    </hint>
    <hint type="destinationlabel">
     <x>570</x>
     <y>334</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
